### PR TITLE
Added the label commenter workflow

### LINKED ISSUE #3: [CI] add label commenter workflow
**Description:**

This repository should have the label commenter workflow.
It would run whenever an issue/pr is labelled with one of the targeted labels and comment down any relevant information.

**Contributor Reference:**

The workflow would use: https://github.com/peaceiris/actions-label-commenter

A sample workflow is defined in: https://github.com/meshery/meshery/blob/master/.github/workflows/label-commenter.yml.

It would also need a sample configuration which is defined in: https://github.com/meshery/meshery/blob/master/.github/label-commenter-config.yml

**NOTE:**

- In the above configuration https://github.com/meshery/meshery/blob/88d1c7c559259775ce2affab2a15a32c673c3f52/.github/label-commenter-config.yml#L26-L41 are specific to Meshery and won't be needed on every repository.
- Do comment in PR if you find any label is missing for the repo.

### PR DESCRIPTION
**Pull request by Anubhavdevv**
**Description**
Added label commenter workflow.
It would run whenever an issue/pr is labeled with one of the targeted labels and comment down any relevant information.This 

PR fixes #3 

**Notes for Reviewers**
Added the Label commenter workflow

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/label-commenter-config.yml
Comment (aminoxix) [inline .github/label-commenter-config.yml:17]: @warunicorn19: Shall we need `issue/dco` label here?

**Review by aminoxix — CHANGES_REQUESTED** [review]
@anubhav201241: Thanks for your contribution, we'll consider this under Hacktoberfest, hope you're taking part in such an amazing Open source event. :slightly_smiling_face: 

Just need some clean check from Aditya, to move forward!

File: .github/label-commenter-config.yml
Comment (warunicorn19) [reply .github/label-commenter-config.yml:17]: yeah we could just let it be there, I'll create the label.

**Review by aminoxix — APPROVED** [review]
@anubhav201241: Some changes need to configure, as mentioned below! :slightly_smiling_face:

**Comment by Anubhavdevv** [comment]
thank you @amino19

File: .github/label-commenter-config.yml
Comment (aminoxix) [inline .github/label-commenter-config.yml]: @anubhav201241: We don't need to trigger `issue/dco` workflow for the issue, pl remove [these lines](https://github.com/layer5io/discuss-board/blob/c6315659080f6b0db884b91ad9cbd7dccf511fd2/.github/label-commenter-config.yml#L24-L28).

**Review by aminoxix — CHANGES_REQUESTED** [review]

**Review by aminoxix — APPROVED** [review]
@anubhav201241: Super jet speed, thanks! 
Now, looks good! :partying_face:

**Comment by Anubhavdevv** [comment]
> @anubhav201241: Super jet speed, thanks! Now, looks good! 🥳

Thank you ☺

File: .github/label-commenter-config.yml
Comment (sayantan1413) [inline .github/label-commenter-config.yml]: We do not need this label in this repository.

File: .github/label-commenter-config.yml
Comment (sayantan1413) [inline .github/label-commenter-config.yml]: We do not this need this labels in this repo. This labels are specific to Meshery and some frontend repos.

**Review by sayantan1413 — CHANGES_REQUESTED** [review]
Good work @anubhav201241 !! . Some minor changes are requested.

File: .github/label-commenter-config.yml
Comment (sayantan1413) [reply .github/label-commenter-config.yml]: Hey @amino19 any reason why are we moving this issue body because we do have this same kind of structure in Meshery https://github.com/meshery/meshery/blob/master/.github/label-commenter-config.yml#L23-L28. And also I have verified that if this issue body is missing then the github action bot do not trigger in the PR.

File: .github/label-commenter-config.yml
Comment (Anubhavdevv) [reply .github/label-commenter-config.yml]: sure @sayantan1413 thank you for reviewing I will make the changes

Comment (Anubhavdevv) [reply .github/label-commenter-config.yml]: Hey, @sayantan1413 I have made the changes please review it

File: .github/label-commenter-config.yml
Comment (Anubhavdevv) [reply .github/label-commenter-config.yml]: Hey, @sayantan1413 I have made the changes please review it

**Comment by Anubhavdevv** [comment]
@sayantan1413 Now you can review it

**Comment by Anubhavdevv** [comment]
@warunicorn19 and @sayantan1413 Any more changes?

File: .github/label-commenter-config.yml
Comment (sayantan1413) [reply .github/label-commenter-config.yml]: Hey @anubhav201241 I think we can keep these lines.

**Comment by sayantan1413** [comment]
@anubhav201241 I have requested a small change above then we are good to go.

File: .github/label-commenter-config.yml
Comment (aminoxix) [reply .github/label-commenter-config.yml]: @sayantan1413: I think we need this for pull requests only to get `signed-off`. Is there any issue that needs to be label as `issue/dco`?

Comment (sayantan1413) [reply .github/label-commenter-config.yml]: I was thinking to keep it consistent with Meshery but what you are suggesting also makes sense!

**Review by sayantan1413 — APPROVED** [review]
LGTM! @anubhav201241 🚀

**Comment by Anubhavdevv** [comment]
Thank You @sayantan1413 
When will my first PR be going to merged ? 😅

**Comment by Anubhavdevv** [comment]
@warunicorn19 Any other changes?

**Comment by aminoxix** [comment]
> @warunicorn19 Any other changes?

Nope, just wait for the rest of the maintainers to review & get this merge! :slightly_smiling_face:

**Comment by Anubhavdevv** [comment]
> , just wait for the rest of the main

Thank you @amino19 Hope it will merge 🙌

**Review by warunicorn19 — APPROVED** [review]
Thank you @anubhav201241

**Comment by Anubhavdevv** [comment]
Thank You So much @amino19, @warunicorn19, and @sayantan1413 🙌🔥
Thanks a lot for guiding and reviewing it, It was my first contribution to the Layer 5 Community 
Thanks, A Lot ☺🙌

**Pull request merged**
